tiledb/api/src/config.rs

```diff
@@ -294,6 +294,21 @@ impl Iterator for ConfigIterator<'_> {
 pub enum CommonOption {
     /// Sets an AES256GCM encryption key.
     Aes256GcmEncryptionKey(Vec<u8>),
+
+    // URL for REST server to use for remote arrays.
+    RestServerAddress(String),
+
+    /// Username for login to REST server.
+    RestUsername(String),
+
+    /// Password for login to REST server.
+    RestPassword(String),
+
+    /// Authentication token for REST server (used instead of username/password).
+    RestToken(String),
```

Comment:
Do we know if the token is always valid UTF-8?  If it isn't then `Vec<u8>` would be better

Reply:
Okay it is. From John Kerl: `it's always base64 so yes, & furthermore it's valid ASCII`.